<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>radiografiaui</class>
 <widget class="QWidget" name="radiografiaui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="fondo" native="true">
     <property name="styleSheet">
      <string notr="true">#fondo{
border-image:url(:/Niveles/Nivel5/fondoscanner.png)
}</string>
     </property>
     <widget class="QStackedWidget" name="cuerpo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>201</width>
        <height>291</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="frente">
       <property name="styleSheet">
        <string notr="true">#frente{
border-image: url(:/Niveles/Nivel5/Frente.png)
}</string>
       </property>
       <widget class="QLabel" name="frente2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="frente3">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>130</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="frente4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>250</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="frente5">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>200</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="frente1">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>110</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="espalda">
       <property name="styleSheet">
        <string notr="true">#espalda{
border-image:url(:/Niveles/Nivel5/Espalda.png)}</string>
       </property>
       <widget class="QLabel" name="espalda2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="espalda3">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>100</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="espalda4">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>150</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="espalda5">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>220</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="espalda1">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>260</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="voltear1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="voltear2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>340</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="botonsalir">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
